spec/models/ext_management_system_spec.rb

```diff
@@ -396,6 +396,91 @@
     end
   end
 
+  context "#pause!" do
+    it 'disables an ems with child managers and moves parent to maintenance zone' do
+      zone  = FactoryGirl.create(:zone)
+      ems   = FactoryGirl.create(:ext_management_system, :zone => zone)
+      child = FactoryGirl.create(:ext_management_system, :zone => zone)
+      ems.child_managers << child
+
+      ems.pause!
+
+      expect(ems.enabled).to be_falsy
+      expect(ems.zone).to eq(Zone.maintenance_zone)
+      expect(ems.zone_before_pause).to eq(zone)
+
+      child.reload
+      expect(child.enabled).to be_falsy
```

Comment:
same here

Reply:
Also `expect(child.zone).to eq(Zone.maintenance_zone)`

Reply:
no we don't. I don't remember why, but it was requested above not to do it

Reply:
yeah, we need to understand why that was decided... it seems flawed to change a provider's zone but not it's managers.  They're all external management systems under the covers so we should set the zone.

Reply:
Yeah we should be putting the children into maintenance zone also

Reply:
https://github.com/ManageIQ/manageiq/pull/17452#issuecomment-432958273

Child managers are putting the children to maintenance zone, but it's not part of this PR, it exists in current ManageIQ and it's not included to ExtManagementSystem's base class but to concrete managers. So it's true in real situations (i should have it's own specs) but not with this basic EMS.

Reply:
We should still add a test for it though since it is a non-obvious dependency.

Reply:
I can add test for Cloud manager -> Network child, it's in core, but everything else are included in provider repos, so I'll mention it to that spec